cmd/convert.go

```diff
@@ -93,6 +94,9 @@ func init() {
 
 	// Standard between the two
 	convertCmd.Flags().BoolVarP(&ConvertYaml, "yaml", "y", false, "Generate resource files into yaml format")
```

Comment:
once convertyaml is removed this should still be functional for sometime, so if this flag is set what can be done here is set ConvertJson to be false.